pkg/cryptoauthlib/include/atca_params.h

```diff
@@ -57,24 +74,65 @@ extern "C" {
 #define ATCA_DEVTYPE            (ATECC608A)
```

Comment:
I believe `ATCA_I2C_ADDR`, `ATCA_RX_RETRIES`, and `ATCA_DEVTYPE` should be configurable per device

Reply:
I removed the second device, so this is not relevant anymore, is it?